src/select.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import JWSelect from 'react-select';
+import classNames from 'classnames';
+import identity from './helpers/identity';
+
+export default class Select extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: props.defaultValue
```

Comment:
`defaultValue` sets the initial value of the field on mount and thats it. `value` will perpetually control the value of the field. 

```
<input1 defaultValue="1" />
<input2 value="1" />
```

For `input1` if we type in the field, the field will update. If we type in the field for `input2`, it will remain "1" forever.

The constructor should take into account the possibility of both these values existing / not existing.